utils/doclint/check_public_api/index.js

```diff
@@ -371,6 +371,13 @@ function compareDocumentations(actual, expected) {
           expectedName: 'ClickOptions',
         },
       ],
+      [
+        'Method ElementHandle.clickablePoint() options',
```

Comment:
```suggestion
        'Method ElementHandle.clickablePoint() offset',
```

Reply:
I think this should resolve the doc lint error.

Reply:
wow, thanks!